Add instructions for enabling `linea-sequencer` plugin (#680)
Consensys/doc.linea: docs/developers/guides/run-a-node/use-binary.mdx

</Tabs>

:::warning[Important: enable linea-sequencer]

> You must install and enable the `linea-sequencer` Besu plugin if you are offering RPC services or 
> hosting nodes for usage. Enabling the plugin is a requirement for using certain endpoints, such as 
> [`linea_estimateGas`](/developers/reference/api/linea-estimategas).

Reviewer: ```suggestion
You must install and enable the `linea-sequencer` Besu plugin if you are offering RPC services or 
hosting nodes. Enabling the plugin is a requirement for using certain API methods, such as 
[`linea_estimateGas`](/developers/reference/api/linea-estimategas).
```

> Please see our instructions [here](./linea-sequencer) for installing the plugin.

Reviewer: ```suggestion
Please see our [instructions for installing the plugin](./linea-sequencer).
```

If you are running a node to follow the chain activity, rather than submit transactions, you don't 
need to install `linea-sequencer`.

:::

> 

Reviewer 2: Remove goerli from https://doc-zk-evm-git-linea-sequencer-infura-web.vercel.app/developers/guides/run-a-node/use-binary

Reviewer 2: Remove full stop at end of Step 4

Author: Good spot on both counts, thank you

> If you are simply running a node to follow the chain and its activity, rather than using it to 
> submit transactions, you do not need to install `linea-sequencer`.

Reviewer: ```suggestion
If you are running a node to follow the chain activity, rather than 
submit transactions, you don't need to install `linea-sequencer`.
```
